### ISSUE TITLE
extraQualities, extraFormats and extraFeatures should not expand to the same IRI

### ISSUE THREAD

**Issue by lutzhelm** [IIIF/api#1784]
The `extraQualities`, `extraFormats` and `extraFeatures` properties of the version 3 image information response all expand to the same IRI `http://iiif.io/api/image/3#quality`. This will make it difficult to use (and especially distinguish between) them in hypothetical JSON-LD purposes. As I understand it, it will be impossible to keep them apart during framing (except by the `@vocab` definition in `extraFeatures`) or after they have been expanded.
Is there any reason / advantage behind using the same IRI or maybe a way around the problem I described?

**Comment by zimeon** [comment]
Relevant section in https://iiif.io/api/presentation/3/combined-context.json is:

```

"extraQualities": {
  "@id": "image:quality",
  "@container": "@set"
},
"extraFormats": {
  "@id": "image:quality",
  "@container": "@set"
},
"extraFeatures": {
  "@id": "image:quality",
  "@type": "@vocab",
  "@container": "@set"
},
```

which has been only slightly modified in [current draft for RC2](https://github.com/IIIF/api/blob/master/source/api/presentation/3/combined-context.json#L238-L250).

If I recall, the discussion about merging everything into `quality` was that there was no possibility of overlap and it avoided more term pollution. However, we do want to be able to use a JSON-LD frame to create a valid Image API `info.json` so I think it would make sense to split them, perhaps into `qualities`, `formats` and `features`.

**Comment by azaroth42** [comment]
Agree that we should split into three. I think should be singular, as the property doesn't take an rdf:List as the object.

Thus:  `extraFeatures` would expand to `image:feature`

**Comment by zimeon** [comment]
Editors agree: `image:quality`, `image:format`, `image:feature`

**Comment by zimeon** [comment]
Closed by merge of #1810
